doc/classes/Image.xml

```diff
@@ -761,16 +766,22 @@
 			Represents the size of the [enum CompressMode] enum.
 		</constant>
 		<constant name="USED_CHANNELS_L" value="0" enum="UsedChannels">
+			The image only uses one channel for luminance (black and white).
```

Comment:
Maybe? "black and white" outside of photography could sound literally like just full black or full white pixels.
```suggestion
			The image only uses one channel for luminance (grayscale).
```